realm/realm-library/src/testObjectServer/kotlin/io/realm/BsonTest.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm
+
+import io.realm.internal.util.BsonConverter
+import org.bson.*
+import org.bson.types.Decimal128
+import org.bson.types.ObjectId
+import org.junit.Assert.assertEquals
+import org.junit.Ignore
+import org.junit.Test
+import kotlin.test.assertFailsWith
+
+class BsonTest {
+
+    /**
+     * Simple test to verify semantics of org.bson JSON encoding and decoding.
+     */
+    @Ignore("Only for Bson API evaluation, not testing Realm functionality")
```

Comment:
Why ignore it?

Reply:
I wasn't even sure if it ends up representing the way we use their API, so did not really see a reason for having it. But I can included it.

Reply:
I was just wondering. I didn't look closely at the implementation. If it isn't going to be used it should probably just be removed instead.